packages/airnode-node/src/evm/verification/request-verification.ts

```diff
@@ -13,100 +12,85 @@ export const isValidSponsorWallet = (hdNode: ethers.utils.HDNode, sponsor: strin
 
 // TODO: Remove this once https://api3dao.atlassian.net/browse/AN-400 is done
 export function verifySponsorWallets<T>(
-  requests: Request<T>[],
+  unverifiedRequests: Request<T>[],
   masterHDNode: ethers.utils.HDNode
 ): LogsData<Request<T>[]> {
-  const logs: PendingLog[] = [];
+  const { logs, requests } = unverifiedRequests.reduce(
```

Comment:
@andreogle  - after the long `filter` vs. `reduce` discussion last time, I now agree `reduce` is the safer, more consistent approach  😅

Reply:
haha it grows on you over time - at least it did for me. It can definitely be less readable though